client/routes/roomRoute.coffee

```diff
@@ -48,6 +48,8 @@ openRoom = (type, name) ->
 					$('.message-form .input-message').focus()
 				, 100
 
+			# update user's room subscription
+			Meteor.call 'openRoom', room._id
```

Comment:
Update subscription every time user enter in a room’s route don't seems to be a good idea.